README.md

```diff
@@ -364,6 +364,26 @@ rm ./examples/sample-key
 open-attestation deploy document-store "My Name" --network ropsten --key 0000000000000000000000000000000000000000000000000000000000000003
 ```
 
+## Config file
+
+This command will generate a config file with sandbox DNS, document store and token registry.
+
+Please note that a template config file and a wallet.json file must be provided in order for this command to work.
+
+Please also note that there must have funds in the Ropsten wallet, as we require to pay some gas fees to generate the config file.
+
+Example:
+
+```
+open-attestation config create --output-dir ./examples/config --encrypted-wallet-path /path/to/wallet.json --config-template-path /path/tp/config.json
+```
+
+You will need to provide the `--output-dir` option to specify in which folder the config file will be created in.
```

Comment:
let's make this a list, it's more readable. For instance:
```
- `--output-dir` option specify in which folder the config file will be created in.
- `--encrypted-wallet-path` option indicates a path to a [encrypted wallet](https://docs.ethers.io/v5/api/signer/#Wallet-encrypt).
- `--config-template-path` option to provide a path to a config file.
```

Reply:
ok